dev/MRTCore/mrt/Microsoft.Windows.ApplicationModel.Resources/src/ResourceCandidate.h

```diff
@@ -33,7 +33,7 @@ struct ResourceCandidate : ResourceCandidateT<ResourceCandidate>
     Microsoft::Windows::ApplicationModel::Resources::ResourceContext m_resourceContext = nullptr;
     MrmMapHandle m_resourceMapHandle = nullptr;
     uint32_t m_resourceIndex = static_cast<uint32_t>(-1);
-    hstring m_resourceId;
+    hstring m_resourceId{};
```

Comment:
is this necessary?

Reply:
Thought I saw a prefast on this one but I don't see it anymore so probaly not